### DISCUSSION TITLE
Need just a bit of help.

### DISCUSSION THREAD

**Discussion by jltnol** [tagyoureit/nodejs-poolController#437] [General]
So I HOPE I'm asking all this in the right place but am so far down the rabbit hole, I'm not even sure of that.  I'm trying to get Homebridge Poolcontroller to work in Homebridge.  The download is straight forward, but apparently I need some additional software to make this actually work, starting with nodes-poolController.  I AM on a M-1 Mac, but have downloaded the Node.js 16.15, which included npm 8.5.5. 

node -v returns 16.15.0
and
npm -v returns 8.9.0

I am then tasked with CD into nodejs-poolController to install the , but have no idea where that directly is, and searching, at least with Spotlight doesn't help at all.  

Past this issue, I'd also like to install a local client just for checking to make sure at least I've got this part working correctly (but am not sure if that is even possible) before moving on to getting all this to work in Homebridge.

But for now, just trying to find/locate the 'nodejs-poolController' directory would be a huge help.

**Comment by rstrouse** [comment]
`cd ~/nodejs-poolController` if you cloned the directory from the home directory (most likely).
